topostats/default_config.yaml

```diff
@@ -1,7 +1,10 @@
+# Config file generated 2024-07-29 11:11:05
+# # For more information on configuration and how to use it:
+# https://afm-spm.github.io/TopoStats/main/configuration.html
 base_dir: ./ # Directory in which to search for data files
 output_dir: ./output # Directory to output results to
 log_level: info # Verbosity of output. Options: warning, error, info, debug
-cores: 2 # Number of CPU cores to utilise for processing multiple files simultaneously.
+cores: 1 # Number of CPU cores to utilise for processing multiple files simultaneously.
```

Comment:
Is there a reason for decreasing the number of cores? Roughly doubles processing time and most modern systems have at least 4 cores (although 2 is the default so that a) people's computers don't get clobbered; b) GitHub Runners have only 2 cores).

Reply:
Might have been an accident, or intentional so users can see which file throws an error. Who's on the blame? If it's me, this was probs an accident I think I'd prefer the default to be speedier.

Reply:
Sylvia has just done this too 👍